plugins/topdesk/plugin.spec.yaml

```diff
@@ -0,0 +1,2545 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: topdesk
+title: TopDesk
+description: TopDesk is a cloud/on-premise ticketing system that focuses on ease of use and extensibility
```

Comment:
Should this mention cloud only? 
Does cloud_ready: true and hub tag keyword cloud_enabled need added?

Reply:
@dlaverty-r7 This description is from the ticket for this plugin, but if TopDesk is cloud only then we will correct the description and update the plugin spec.